Formula/i/inchi.rb

```diff
@@ -0,0 +1,95 @@
+class Inchi < Formula
+  desc "IUPAC International Chemical Identifier"
+  homepage "https://www.inchi-trust.org/"
+  url "https://github.com/IUPAC-InChI/InChI/releases/download/v1.07.1/INCHI-1-SRC.zip"
+  sha256 "fe6e1ee25714988f7b86420b7615b4e1d7c01fda9b93d63b634a0c021ac9f917"
+  license "MIT"
+
+  # These used to be part of open-babel
+  link_overwrite "include/inchi/inchi_api.h", "lib/libinchi.dylib", "lib/libinchi.so"
+
+  # Fix dylib file names
+  # PR ref: https://github.com/IUPAC-InChI/InChI/pull/62
+  patch :p2, :DATA
+
+  def install
+    bin.mkpath
+    lib.mkpath
+
+    args = ["C_COMPILER=#{ENV.cc}", "BIN_DIR=#{bin}", "LIB_DIR=#{lib}"]
+    system "make", "-C", "INCHI_API/libinchi/gcc", *args
+    system "make", "-C", "INCHI_EXE/inchi-1/gcc", *args
+
+    # Add major versioned and unversioned symlinks
+    libinchi = shared_library("libinchi", version.to_s[/^(\d+\.\d+)/, 1])
```

Comment:
```suggestion
    libinchi = shared_library("libinchi", version.major_minor)
```
Isn't this equivalent?

Reply:
It doesn't retain zeros so ends up with `1.7`.

Reply:
Even if manually specify `version` in the formula? Huh.

Reply:
It's mainly during the processing of the `version` to alternate forms:
```
brew(main):001> Formula["inchi"].version.to_s
=> "1.07.1"
brew(main):002> Formula["inchi"].version.major_minor.to_s
=> "1.7"
brew(main):003> Formula["inchi"].version.major_minor_patch.to_s
=> "1.7.1"
```

Reply:
Seems like a bug in `brew`.